packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -347,13 +354,20 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
         const assetID = v4();
         const { filename, filepath, stats, stream } = asset;
 
-        await safePush({ action: 'start', assetID, data: { filename, filepath, stats } });
+        await safePush(
+          {
+            action: 'start',
+            assetID,
+            data: { filename, filepath, stats },
+          },
+          callback
+        );
 
         for await (const chunk of stream) {
-          await safePush({ action: 'stream', assetID, data: chunk });
+          await safePush({ action: 'stream', assetID, data: chunk }, callback);
```

Comment:
Won't this call `callback` twice in the case of an error?

Reply:
I don't know if it would, anyways I added some changes that would prevent such a possibility